### ISSUE TITLE
Automated lighthouse check

### ISSUE THREAD

**Issue by asvae** [epicmaxco/vuestic-ui#2853]
- [ ] It should be integrated into release flow
- [ ] Test should show results in concise way + fail on bad results (below green?)

**Comment by m0ksem** [comment]
Looks like netlify does it.
